### DISCUSSION TITLE
forum-session-Deaf-Hub

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/accessibility2024#130] [General]
# forum-session-Deaf-Hub

Join Deaf Hub colleagues and friends to learn more about the deaf and deaf-blind experience and how to give and get support navigating academic settings.

https://accessibility2024.arxiv.org/forum-session-Deaf-Hub

**Comment by SBBCornell** [comment]
Welcome everyone! Please share thoughts and resources of interest to the community.
